win32ss/drivers/videoprt/int10.c

```diff
@@ -23,11 +23,61 @@
 
 #include <ndk/kefuncs.h>
 #include <ndk/halfuncs.h>
+#include <ndk/mmfuncs.h>
 
 #define NDEBUG
 #include <debug.h>
 
 /* PRIVATE FUNCTIONS **********************************************************/
+#define IsLowV86Mem(_Seg, _Off) ((((_Seg) << 4) + (_Off)) < (0xa0000))
```

Comment:
Please add a separator newline between the code and the "code file section title" ^^
```suggestion

#define IsLowV86Mem(_Seg, _Off) ((((_Seg) << 4) + (_Off)) < (0xa0000))
```